.github/workflows/jira-issue-create.yml

```diff
@@ -0,0 +1,21 @@
+# Creates jira tickets for new github issues to help triage
+name: Jira Issue Creator For ampli-examples
+
+on:
+  issues:
+    types: [opened]
+  workflow_call:
+    inputs:
+      label:
+        type: string
+
+jobs:
+  call-workflow-passing-data:
+    uses: amplitude/Amplitude-TypeScript/.github/workflows/jira-issue-create-template.yml@main
+    with:
+      label: "ampli"
```

Comment:
Minor, maybe "ampli examples"? I think we have another repo https://github.com/amplitude/ampli.

Reply:
nvm, think we are good